src/neuroconv/basedatainterface.py

```diff
@@ -24,6 +27,36 @@ def get_conversion_options_schema(cls):
     def __init__(self, **source_data):
         self.source_data = source_data
 
+        # Time-related attributes
+        self._start_time: Optional[datetime] = None
+        self._relative_start_time: Optional[Timedelta] = None
+        self._rate: Optional[float] = None
+        # self._times: Optional[ArrayType] = None  # TODO: generalize to vector of timestamps
+
+    def set_start_time(self, timestamp: datetime):
+        """Set a timestamp as a datetime object to indicate the exact start time of the series in the DataInterface."""
+        self._start_time = timestamp
+
+    def get_start_time(self) -> Optional[datetime]:
+        """Retrieve the timestamp which indicates the exact start time of the series in the DataInterface."""
+        return self._start_time
+
+    def set_relative_start_time(self, time_delta: Timedelta):
+        """Set a Timedelta object to indicate the relative start time of the series in the DataInterface."""
+        self._relative_start_time = time_delta
```

Comment:
I think it might be easier to just do this in seconds. I don't expect there to be really a really large time difference of hours or more that would require the use of `timedelta`